client/src/lib/types.ts

```diff
@@ -0,0 +1,19 @@
+export type Location = 'bananaBunches' | 'healthyTown' | 'dairyPark'
```

Comment:
You can also use enums for things like this

Reply:
ok gimme 10 min